test/date-time/date-time-create.test.ts

```diff
@@ -0,0 +1,1068 @@
+            {
+                Assert.ok(new DateTime({ value: validValue, zone: "UTC-3" }));
+            }
+        );
+
+        test(`Given valid value and zone as valid UTC offset +14:00
+        when the constructor is called 
+        then it should return a DateTime object`,
+            () =>
+            {
+                Assert.ok(new DateTime({ value: validValue, zone: "UTC+14:00" }));
+            }
+        );
+
+        test(`Given valid value and zone as valid UTC offset -12:00
+        when the constructor is called 
+        then it should return a DateTime object`,
+            () =>
+            {
+                Assert.ok(new DateTime({ value: validValue, zone: "UTC-12:00" }));
+            }
+        );
+
+        test(`Given valid value and zone as valid UTC offset +00:01
+        when the constructor is called 
+        then it should return a DateTime object`,
+            () =>
+            {
+                Assert.ok(new DateTime({ value: validValue, zone: "UTC+00:01" }));
+            }
+        );
+
+        test(`Given valid value and zone as valid UTC offset -00:01
+        when the constructor is called 
+        then it should return a DateTime object`,
+            () =>
+            {
+                Assert.ok(new DateTime({ value: validValue, zone: "UTC-00:01" }));
+            }
+        );
+
+        test(`Given valid value and zone as invalid UTC offset +14:01
+        when the constructor is called 
+        then it should throw a validation error`,
+            () =>
+            {
+                checkIsInvalid(validValue, "UTC+14:01"); // max is +14:00
+            }
+        );
+
+        test(`Given valid value and zone as invalid UTC offset -12:01
+        when the constructor is called 
+        then it should throw a validation error`,
+            () =>
+            {
+                checkIsInvalid(validValue, "UTC-12:01"); // min is -12:00
+            }
+        );
+
+        test(`Given valid value and zone as invalid UTC offset +15
+        when the constructor is called 
+        then it should throw a validation error`,
+            () =>
+            {
+                checkIsInvalid(validValue, "UTC+15"); // max is +14:00
+            }
+        );
+
+        test(`Given valid value and zone as invalid UTC offset -13
+        when the constructor is called 
+        then it should throw a validation error`,
+            () =>
+            {
+                checkIsInvalid(validValue, "UTC-13"); // max is -12:00
+            }
+        );
+    }
+    );
+
+
+    suite("Now", () =>
+    {
+        test(`Given DateTime.now() in default zone (utc)
+        when it's compared to Luxon DateTime now in utc
+        then it should be same`,
+            () =>
+            {
+                // this might fail if at the end of minute and next is at start of minute. but rare condition
+                Assert.strictEqual(DateTime.now().value, LuxonDateTime.utc().toFormat("yyyy-MM-dd HH:mm"));
```

Comment:
yeeea i don't think you need to test this

Reply:
unless you want to temporarily replave Date.now to return what you want it to return. it so it always exact

Reply:
lets talk about these